### ISSUE TITLE
A big sinking glitch

### ISSUE THREAD

**Issue by danielcbailey** [msummers123/Movecraft-3#165]
When a craft sinks, it will keep moving down until go further. That is the idea at least. The truth is, crafts STILL sink into infinity, just much harder to find. When I was testing my part submerged vertical cannons on the Airship Pirates Public Build Server, they would shoot a ship down successfully. However the sunken ship would replace all non solid blocks, redstone, air, repeaters, anything with water for beneath the ocean. Why this is important? A sinking craft completely destroys anything it sinks on. These non solid blocks also have a shield keeping them 'safe', but the plugin still replaces it with water.

**Comment by msummers123** [comment]
Yes, this behavior is by design. Being underneath a sinking ship is unhealthy :)

**Comment by msummers123** [comment]
Wait, I may have misunderstood. Are you saying the blocks in question had a solid block in between them and the sinking ship?

**Comment by danielcbailey** [comment]
Anything from 1-10 blocks was in my "test", more or less frowning when my cannon implodes just to find out that EVERY redstone or air block was water.

**Comment by keough99** [comment]
I may have noticed it on my server and never thought anything of it. Hmmm I'll have to do rigorous testing to find out.
